velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -563,6 +565,84 @@ TpchPlan TpchQueryBuilder::getQ10Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ12Plan() const {
+  std::vector<std::string> ordersColumns = {"o_orderkey", "o_orderpriority"};
+  std::vector<std::string> lineitemColumns = {
+      "l_receiptdate",
+      "l_orderkey",
+      "l_commitdate",
+      "l_shipmode",
+      "l_shipdate"};
+
+  auto ordersSelectedRowType = getRowType(kOrders, ordersColumns);
+  const auto& ordersFileColumns = getFileColumnNames(kOrders);
+  auto lineitemSelectedRowType = getRowType(kLineitem, lineitemColumns);
+  const auto& lineitemFileColumns = getFileColumnNames(kLineitem);
+
+  auto planNodeIdGenerator = std::make_shared<PlanNodeIdGenerator>();
+  core::PlanNodeId ordersScanNodeId;
+  core::PlanNodeId lineitemScanNodeId;
+
+  const auto receiptDate = "l_receiptdate";
```

Comment:
These seem to be used in only one place below. Why not define them in-place to the formatDateFilter call itself ?